aspnetcore/fundamentals/error-handling/includes/error-handling8.md

```diff
@@ -78,9 +78,9 @@ The following code uses a lambda for exception handling:
 
 ## IExceptionHandler
 
-[IExceptionHandler](https://source.dot.net/#Microsoft.AspNetCore.Diagnostics/ExceptionHandler/IExceptionHandler.cs,adae2915ad0c6dc5) is an interface that gives the developer a callback for handling known exceptions in a central location.
+[IExceptionHandler](/dotnet/api/microsoft.aspnetcore.diagnostics.iexceptionhandler) is an interface that gives the developer a callback for handling known exceptions in a central location.
```

Comment:
Now that the xref helper tool is available, we can easily generate xref links. I see some xref links here and some URLs -- what determines which is used in each instance?

Reply:
I agree on the confusion on some of these. Shouldn't this be ...

```suggestion
<xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandler> is an interface that gives the developer a callback for handling known exceptions in a central location.
```

Reply:
BTW ... sidenote on the Blazor WASM Xref Gen tool ...

Because it's using a CORS proxy to get the API Browser pages in order to pull the `ms.assetid`s to make links, there is a noticeable delay while it's making its look-ups. The CORS proxy is kind'a on the slow side 🐌. I'll work on that. A few avenues are ...

* Reduce the number of initial look-ups and/or page the results in smaller batches.
* Use Blazor's Streaming Rendering feature to render them as they are generated.

I also note that I have the font size too large for the results, so I'm going to knock it down a bit to get more results on the screen.